--- httpd.conf.orig     2018-02-01 20:14:27.199360839 +0000
+++ httpd.conf  2018-02-01 20:15:43.731366020 +0000
@@ -63,8 +63,8 @@
 # Example:
 # LoadModule foo_module modules/mod_foo.so
 #
-LoadModule mpm_event_module lib/httpd/mod_mpm_event.so
-#LoadModule mpm_prefork_module lib/httpd/mod_mpm_prefork.so
+#LoadModule mpm_event_module lib/httpd/mod_mpm_event.so
+LoadModule mpm_prefork_module lib/httpd/mod_mpm_prefork.so
 #LoadModule mpm_worker_module lib/httpd/mod_mpm_worker.so
 LoadModule authn_file_module lib/httpd/mod_authn_file.so
 #LoadModule authn_dbm_module lib/httpd/mod_authn_dbm.so
@@ -170,7 +170,8 @@
 #LoadModule speling_module lib/httpd/mod_speling.so
 #LoadModule userdir_module lib/httpd/mod_userdir.so
 LoadModule alias_module lib/httpd/mod_alias.so
-#LoadModule rewrite_module lib/httpd/mod_rewrite.so
+LoadModule rewrite_module lib/httpd/mod_rewrite.so
+LoadModule php7_module lib/httpd/mod_php7.so
 
 <IfModule unixd_module>
 #
@@ -257,7 +258,7 @@
     # It can be "All", "None", or any combination of the keywords:
     #   AllowOverride FileInfo AuthConfig Limit
     #
+    AllowOverride All
-    AllowOverride None
 
     #
     # Controls who can get stuff from this server.
@@ -270,7 +271,7 @@
 # is requested.
 #
 <IfModule dir_module>
-    DirectoryIndex index.html
+    DirectoryIndex index.html index.php
 </IfModule>
 
 #
@@ -523,3 +524,6 @@
 SSLRandomSeed connect builtin
 </IfModule>
 
+<FilesMatch \.php$>
+SetHandler application/x-httpd-php
+</FilesMatch>

